src/main/java/com/patchworkmc/Patchwork.java

```diff
@@ -305,6 +293,25 @@ private void writeLogo(JsonObject json, FileSystem fs) throws IOException {
 		}
 	}
 
+	private void generateDevJar(Path relativeJarPath) {
```

Comment:
Technically there can be more than one dev jar.
```suggestion
	private void generateDevJars(Path relativeJarPath) {
```

Reply:
But invoking that method once generates one dev jar

Reply:
In the case of multiple devMappings providers it would generare multiple output jars.

Reply:
Currently, the gui cannot specify multiple dev mappings. Are you going to make it generate multiple dev jars for one mod with different yarn versions?

Reply:
Perhaps, as it is the API does support multiple dev jars so the naming should reflect that. Only one dev jar is generated currently though.